CogniteSdk/test/fsharp/Assets.fs

```diff
@@ -468,11 +468,16 @@ let ``Filter assets on single Label is Ok`` () = task {
     // Act
     let! res = writeClient.Playground.Assets.ListAsync query
 
-    let len = Seq.length res.Items
+    let allItemsMatch =
+        res.Items |> Seq.map (fun item ->
+            item.Labels
+            |> Seq.map (fun label -> label.ExternalId)
+            |> Seq.contains "AssetTestLabel1"
+        ) |> Seq.forall id
 
     // Assert
     // Test may fail if datastudio playground data changes
-    test <@ len > 0 @>
```

Comment:
Should probably have the `len > 0` as well since `allItemsMatch` will be true for an empty result:

```fs
> Seq.forall id [];;
val it : bool = true
```